### PR TITLE
HW1: Added project and contribution descriptions in markdown files. Updated proposal markdown file accordingly.

### PR DESCRIPTION
**Pull request by mwsepulveda**
Finished HW1!

- Did not yet come up for a name for my personal project or create a repo for it.

### CODE REVIEW

File: contribution_description.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# freeCodeCamp
```
Comment (timkpaine) [inline contribution_description.md:2]: move to `projects/javascript/freecodecamp.md`

File: project_description.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (timkpaine) [inline project_description.md:1]: move to `projects/javascript/<whatever you want to call it>.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: contribution_description.md (hunk 1 above)
Comment (mwsepulveda) [reply contribution_description.md]: moved to appropriate location

File: project_description.md (hunk 2 above)
Comment (mwsepulveda) [reply project_description.md]: moved to appropriate location with name switch from "js-library" to "noobJS"

**Comment by mwsepulveda** [comment]
Unsure if I or you are the one who should resolve the conversations after the commit that addresses requested changes. Please let me know!

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**